openshift/rh-che.config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: che
+type: Opaque
+data:
+  che-host: "glusterpoc37aws.devshift.net"
+  workspace-storage: "/home/user/che/workspaces"
+  workspace-storage-create-folders: "false"
+  local-conf-dir: "/etc/conf"
+  openshift-serviceaccountname: "che"
+  che-server-evaluation-strategy: "always-external-custom"
+  che.logs.dir: "/data/logs"
+  che.docker.server_evaluation_strategy.custom.template: "<serverName>-<if(isDevMachine)><workspaceIdWithoutPrefix><else><machineName><endif>-<if(workspacesRoutingSuffix)><workspacesRoutingSuffix><else><externalAddress><endif>"
+  che.docker.server_evaluation_strategy.custom.external.protocol: "https"
+  che.predefined.stacks.reload_on_start: "true"
+  log-level: "INFO"
+  docker-connector: "openshift"
+  port: "8080"
+  remote-debugging-enabled: "false"
+  che-oauth-github-forceactivation: "true"
+  workspaces-memory-limit: "1900Mi"
+  workspaces-memory-request: "1100Mi"
+  enable-workspaces-autostart: "false"
+  keycloak-oso-endpoint: "https://sso.openshift.io/auth/realms/fabric8/broker/openshift-v3/token"
+  keycloak-github-endpoint: "https://auth.openshift.io/api/token?for=https://github.com"
+  keycloak-disabled: "false"
```

Comment:
I assume we can remove this one. Keycloak is never disabled in the multi-tenant scenario.

Reply:
@davidfestal and by default keycloak would be enabled without this property, right ?

Reply:
in multi-tenant `rh-che` : yes